src/components/funfacts/funfact-slideshow/FunfactSlideshow.tsx

```diff
@@ -102,9 +102,9 @@ const Carousel: FC<FunfactSlideshowPropsI> = (props) => {
 
 // inspired by: https://codesandbox.io/s/framer-motion-image-gallery-pqvx3?file=/src/Example.tsx:1598-2616
 
-const FunfactSlideshow: React.FC<FunfactSlideshowPropsI> = (funfacts) => {
+const FunfactSlideshow: FC<FunfactSlideshowPropsI> = (funfacts) => {
```

Comment:
Not related to your PR, but maybe we should move one of the components into a separate file?